### PR TITLE
various SLAS adjustments, added callbackURIs

### PR DESCRIPTION
**Pull request by sandragolden**
Mostly just fixing some minor typos in params, also added callback URIs needed for password resets and passwordless login

cc: @hnestmann

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -2062,11 +2064,12 @@ program
         const secret = options.secret;
         const channels = !options.channels || options.channels.split(',').map(item => item.trim());
         const scopes = !options.scopes || options.scopes.split(',').map(item => item.trim());
-        const redirecturis = !options.redirecturis || options.redirecturis.split(',').map(item => item.trim());
+        const redirecturis = options.redirecturis ? options.redirecturis.split(',').map(item => item.trim()) : [];
```
Comment (sandragolden) [inline cli.js:2067]: needs to be set to an empty array if the client does not have redirectURLs (passwordless login)

File: lib/slas.js [hunk 2]
```diff
@@ -79,10 +79,10 @@ const slas = {
                     handleCLIError('Could not get tenant: ', e.message, asJson)
                 }
             },
-            list: async (shortcode, asJson) => {
+            list: async (shortcode, tenantId, asJson) => {
                 let result
                 try {
-                    result = await slas.api.tenant.list(shortcode)
+                    result = await slas.api.tenant.list(shortcode, tenantId)
```
Comment (sandragolden) [inline lib/slas.js:85]: I think the real issue here is that the `get tenants` endpoint doesn't exist anymore but this at least returns a result for 1 single tenant... but i can revert out so we can get the other fixes in

File: lib/slas.js [hunk 3]
```diff
@@ -120,7 +120,7 @@ const slas = {
                     handleCLIError('Could not get tenant: ', e.message, asJson)
                 }
             },
-            list: async (shortcode, tenantId, asJson) => {
+            list: async (tenantId, shortcode, asJson) => {
```
Comment (sandragolden) [inline lib/slas.js:123]: order from cli expects the reverse:
`await slas.cli.client.list(options.tenant, options.shortcode, asJson);`

**Review by clavery — APPROVED** [review]
:+1:

**Comment by sandragolden** [comment]
@tobiaslohr This should be ready to merge. I double checked all the commands via the command line as well as via the JS API and they all work without error with the exception of `slas:tenant:list` and `slas:tenant:delete` which are restricted to staff only / internal role. I wasn't able to get the JS API to work at all because of auth errors so I added `token` as the last parameter for all of the API functions.

I don't think there are any breaking-changes here. I switched the order of the params on `slas.cli.client.list` because that was throwing an error because the params were reversed. I added `callbackuris` to `client:add` which shouldn't be breaking either. All the other updates were errors (like `callbackuris` and `redirecturis` need to always be arrays if null is passed in).

I see this is failing `build-and-test` in CircleCI but I don't think that's from any changes I've made as the same errors look to be thrown in the master branch. Let me know if I need to take a look at that.

**Review by tobiaslohr — APPROVED** [review]

**Comment by tobiaslohr** [comment]
Thanks for your contribution @sandragolden!

**Pull request merged**